upi/openstack/inventory.yaml

```diff
@@ -19,8 +19,8 @@ all:
       # Subnet pool prefix length
       host_prefix: '23'
       # Name of the SDN.
-      # Possible values are OpenshiftSDN or Kuryr.
-      os_networking_type: 'OpenshiftSDN'
+      # Possible values are OVNKubernetes or OpenShiftSDN
+      os_networking_type: 'OVNKubernetes'
```

Comment:
While I agree that OVNKubernetes should now be the default, this is a functional, unrelated, change and possibly belongs to a different PR.

Reply:
Agreed, fixing that now! I believe this thinking can also be applied to your other comment - the fact that no networkType validation is in place is unrelated to Kuryr.

Reply:
Yes, I also think that adding validation for the network type should be its own PR.